model/godleyIcon.cc

```diff
@@ -357,8 +357,8 @@ namespace minsky
       {
         // top justification at bottom of icon if displayed, bottom justified otherwise
         v->rotation(90);
-        v->moveTo(x, y + v->y() - (variableDisplay? v->top(): v->bottom()));
-        x+=v->width();
+        v->moveTo(x,y + v->x() - (variableDisplay? v->left(): v->right()));  // need to use x after stock vars have been rotated. for tickets 1230/1230
+        x+=v->height();
```

Comment:
Currently width/height also refer to variables after rotation, so this also doesn't make sense:
    double Item::rotation(const double& r) {
      m_rotation=r;
      bb.update(*this);
      return m_rotation;
    }

Reply:
Maybe the problem is the bounding box update in the iWidth and iHeight setters? Maybe they shouldn't be there, and only in resize and rotate? I'll try that.

The problem here is that the stocks are positioned relative to the Godly icon and not the canvas background, which is why the stocks were spaced irregularly in ticket 1232.